dts/arm/st/mp13/stm32mp13.dtsi

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-a.dtsi>
+
+#include <freq.h>
+#include <mem.h>
+
+#include <zephyr/dt-bindings/clock/stm32mp13_clock.h>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/interrupt-controller/arm-gic.h>
+#include <zephyr/dt-bindings/reset/stm32mp13_reset.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu0: cpu@0 {
+			compatible = "arm,cortex-a7";
+			device_type = "cpu";
+			reg = <0>;
+			fpu = "enabled";
+		};
+	};
+
+	soc {
+		interrupt-parent = <&gic>;
+
+		sysram: memory0@2ffde000  {
+			compatible = "mmio-sram";
+			reg = <0x2FFDE000 DT_SIZE_K(256)>;
```

Comment:
SYSRAM starts at 0x2FFE0000 and is 128kB large.
It is the 8kB SRAM3 that is aliased at 0x2FFDE000.